doc/nrf/releases_and_maturity/migration_guides.rst

```diff
@@ -3,6 +3,22 @@
 Migration guides
 ################
 
+Required changes
```

Comment:
I guess this page is only TOC. You should put this information in migration_guide_2_6.rst, no?

Reply:
Hmm, I've never added migration steps yet, but adding it to the `migration_guide_2_6.rst` looks better indeed :)